src/Plugin.php

```diff
@@ -78,7 +90,9 @@ public function __invoke(RegistrationInterface $api, SimpleXMLElement $config =
         $api->registerHooksFromClass(ContainerHandler::class);
 
         $stubs = array_merge(
-            $this->getCommonStubs(), $this->getStubsForMajorVersion(Kernel::MAJOR_VERSION)
+            $this->getCommonStubs(),
+            $this->getStubsForMajorVersion(Kernel::MAJOR_VERSION),
+            $this->getFormStubs()
```

Comment:
thus this change can be reverted

Reply:
Done, but I had to move stubs for events one folder up; it doesn't match Symfony folder structure anymore.